code/Model/SiteTree.php

```diff
@@ -734,14 +742,11 @@ public function getAbsoluteLiveLink($includeStageEqualsLive = true)
      */
     public function CMSEditLink()
     {
-        $link = Controller::join_links(
-            CMSPageEditController::singleton()->Link('show'),
-            $this->ID
-        );
-        return Director::absoluteURL($link);
+        // This method has to be implemented here to satisfy the CMSPreviewable interface.
```

Comment:
This should be a method docblock comment rather than an inline comment

Reply:
I disagree - it's a comment for our internal understanding, not something that needs to be highlighted in the IDE.